libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/HistoryAttachmentsUtils.kt

```diff
@@ -60,11 +62,12 @@ internal object HistoryAttachmentsUtils {
 
     fun delete(file: File): Boolean = file.delete()
 
-    fun copyToAndRemove(from: File, filename: String): File {
-        val to = from.copyTo(File(from.parent, filename))
-        from.delete()
-        return to
-    }
+    suspend fun copyToAndRemove(from: File, filename: String): File =
+        withContext(Dispatchers.IO) {
+            val to = from.copyTo(File(from.parent, filename))
+            from.delete()
+            to
```

Comment:
I believe this can be replaced with:
```suggestion
            return File(from.parent, filename).also { from.renameTo(to) }
```
Renaming (which is the same as moving) should be more effective than copying and deleting.

Reply:
It'd be `from.renameTo(File(from.parent, filename))`, right?

Reply:
Nevermind, `from.renameTo(File(from.parent, filename))` returns a boolean not the `File` 👍